test/issue752_test.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('ProductPackaging bug', () {
+    final testProduct = Product();
+    const productName = 'Orange';
+    testProduct.productName = productName;
+    testProduct.packagings = [ProductPackaging()..quantityPerUnit = '75cl'];
+    final productJson = testProduct.toJson();
+    try {
+      final productRestored = Product.fromJson(productJson);
+      expect(productRestored.productName, productName);
+    } catch (e, s) {
+      print(s);
+      fail('Could not restore Product from JSON ${e.toString()}');
```

Comment:
FYI, just printing `$e` is enough. It's implicit to call `toString`

Reply:
Addressed in in commit 5f127c2. Thanks!